sdk/identity/azure-identity/tests/test_imds_credential_async.py

```diff
@@ -267,6 +270,7 @@ async def test_system_assigned_tenant_id(self):
 
     @pytest.mark.usefixtures("user_assigned_identity_client_id")
```

Comment:
If I'm not mistaken, `pytest` is trying to phase out this fixture request syntax. If you wanted to, you could use the new [`recorded_test`](https://github.com/Azure/azure-sdk-for-python/blob/0c156c9d139326703d1a50bad19d42ea6561cc15/tools/azure-sdk-tools/devtools_testutils/proxy_fixtures.py#L29) fixture (for sync and async) instead of the `@recorded_by_proxy` decorator.

Then I think you would be able to do something like `test_user_assigned(recorded_test, user_assigned_identity_client_id)`. Not an urgent issue though, if this works today 🙂